.github/workflows/ci.yaml

```diff
@@ -136,6 +136,8 @@ jobs:
             test: { name: "arch", case: "arch" }
           - backend: kvm
             test: { name: "apertis", case: "apertis" }
+          - backend: kvm
+            test: { name: "run", case: "run" }
```

Comment:
kvm actually has the highest latency atm because of the runner setup; and really this should probably run on all types.. That said could you merge it with an existing tests so we don't need a new CI job for it ? 

e.g. maybe merge it with the recipes test and the overlay test :)